aten/src/ATen/cpu/vec/vec512/vec512_qint.h

```diff
@@ -430,6 +430,11 @@ struct Vectorized<c10::qint8> : public Vectorizedqi {
     // constructor for moving the enum
     Vectorized(const Vectorized<c10::qint8>& other) : Vectorizedqi(other.vals) { }
 
+    // This is added to avoid error: definition of implicit copy assignment operator
+    // for 'Vectorized<c10::qint8>' is deprecated because it has a user-declared
+    // copy constructor [-Werror,-Wdeprecated-copy]
+    Vectorized& operator=(const Vectorized<c10::qint8>&) = default;
```

Comment:
@imaginary-person  can you please take a look if this change is fine.

Reply:
I'd say we should just merge if CI looks fine, we can always revert if this horribly breaks things but regressions should ideally be caught by CI.

Reply:
Sorry for my late response! This change LGTM!

As for the trunk breakage, I suspect it's because of two unexpected successes.
Removing the following two lines should fix this issue for the failing CI check -

https://github.com/pytorch/pytorch/blob/074dc7465e4626eb10dffe8fd44312c4637b9e99/torch/testing/_internal/common_methods_invocations.py#L18999

https://github.com/pytorch/pytorch/blob/074dc7465e4626eb10dffe8fd44312c4637b9e99/torch/testing/_internal/common_methods_invocations.py#L19017

However, I'm curious about what change in this PR caused these two tests to unexpectedly pass in the first place, and if removing the above lines would cause some other CI check to fail.

Thanks!

Reply:
An addendum - `test_nnc_correctness` has been marked with a `slowTest` decorator, and it seems there are only two slow CI checks - one for CPU, and another for CUDA. But the above two tests (`test_nnc_correctness__masked_argmax_cpu_bfloat16` & `test_nnc_correctness__masked_argmin_cpu_bfloat16`  are skipped for CUDA, so removing the above two lines seems to be safe (should not affect other CI checks).

Nevertheless, it'd be good to know why these tests passed with this PR.

Thanks!